### ISSUE TITLE
Client crash opening inventory

### ISSUE THREAD

**Issue by lebanni** [Tschipp/CarryOn#115]
MC 1.12.2
Forge 14.23.4.2708
Carry On 1.9

[crash-2018-06-21_15.18.33-client.txt](https://github.com/Tschipp/CarryOn/files/2123751/crash-2018-06-21_15.18.33-client.txt)

**Comment by caiorasch** [comment]
I've got the same happening, but I don't know if it is related. It was working all well before, but I just added Hammer Core(+ Aether 2, Orbis API and ForgeMultipart) and I saw in your crash log that you also had Hammer Core. Opened the game, tried opening my inventory and bam. VanillaFix pointed to CarryOn so here am I:

MC - 1.12.2 - Forge 14.23.4.2705
Carry On 1.9
Crash Log: https://paste.dimdev.org/eyijosowom.mccrash
